src/NLog/Internal/PropertyHelper.cs

```diff
@@ -98,6 +98,8 @@ internal static void SetPropertyFromString(object obj, string propertyName, stri
             }
             catch (Exception exception)
             {
+                InternalLogger.Warn(exception, "Error when setting property '{0}' on '{1}'", propInfo.Name, obj);
+
                 if (exception.MustBeRethrown())
                 {
                     throw;
```

Comment:
Breaking change: If `LogManager.ThrowExceptions` is `true`, then the exception would not be wrapped as an `NLogConfigurationException` as it would have before. The `if` must probably change to `MustBeRethrownImmediately()`.